samcli/lib/build/workflow_config.py

```diff
@@ -117,6 +117,32 @@ def get_workflow_config(runtime, code_dir, project_dir):
                                           .format(runtime, str(ex)))
 
 
+def supports_build_in_container(config):
```

Comment:
Do we need this method? This seems like configuration for a workflow. Would it be better to store this in the Workflow Config directly? Or even add it to the [runtime_template](https://github.com/awslabs/aws-sam-cli/blob/develop/samcli/local/common/runtime_template.py) class and read it from there?

Reply:
I actually implemented both the changes (add to workflow config & update runtime_template) but neither felt right. 

Workflow Config is essentially a "primary key" that identifies a workflow. Whether a workflow can run in a container or not is an attribute of the workflow, and not a part of the primary key. Hence the new method. 

Technically both the other options can work, but they break the conceptual model IMO.